tests/e2e/tests/smoke/database/add-db-from-welcome-page.e2e.ts

```diff
@@ -1,27 +1,53 @@
-import { rte } from '../../../helpers/constants';
+import { ClientFunction, t } from 'testcafe';
+import { env, rte } from '../../../helpers/constants';
 import { acceptLicenseTerms, addNewStandaloneDatabase, deleteDatabase } from '../../../helpers/database';
 import { MyRedisDatabasePage, AddRedisDatabasePage } from '../../../pageObjects';
 import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+import { deleteAllDatabasesApi } from '../../../helpers/api/api-database';
 
 const myRedisDatabasePage = new MyRedisDatabasePage();
 const addRedisDatabasePage = new AddRedisDatabasePage();
+const getPageUrl = ClientFunction(() => window.location.href);
 
 fixture `Add database from welcome page`
     .meta({ type: 'smoke' })
     .page(commonUrl)
     .beforeEach(async() => {
         await acceptLicenseTerms();
-    })
-    .afterEach(async() => {
-        //Delete database
-        await deleteDatabase(ossStandaloneConfig.databaseName);
+        await deleteAllDatabasesApi();
+        // Reload Page
+        await t.eval(() => location.reload());
     });
 test
+    .after(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    })
     .meta({ rte: rte.standalone })('Verify that user can add first DB from Welcome page', async t => {
-        //Delete all the databases to open Welcome page
-        await myRedisDatabasePage.deleteAllDatabases();
         await t.expect(addRedisDatabasePage.welcomePageTitle.exists).ok('The welcome page title');
         //Add database from Welcome page
         await addNewStandaloneDatabase(ossStandaloneConfig);
         await t.expect(myRedisDatabasePage.dbNameList.withExactText(ossStandaloneConfig.databaseName).exists).ok('The database adding', { timeout: 10000 });
     });
+test
+    .meta({  env: env.web, rte: rte.standalone })('Verify that all the links are valid from Welcome page', async t => {
+        // Verify build from source link
+        await t.expect(addRedisDatabasePage.buildFromSource.visible).ok('Build from source link');
```

Comment:
please change error messages in expects from 'Build from source link' to something like 'Build from source link not visible' and the same above

Reply:
and seems like you don't need these expects for buttons visibility as testCafe check it in the step below in .click() method
await t.click(addRedisDatabasePage.buildFromSource);

Reply:
fixed both